src/Helsenorge.Messaging/Abstractions/IMessageProtection.cs

```diff
@@ -53,14 +54,15 @@ public interface IMessageProtection
         /// <param name="encryptionCertificate">The public key <see cref="X509Certificate2"/> which will be used to encrypt the data.</param>
         /// <param name="signingCertificate">The private key <see cref="X509Certificate2"/>which will be used to sign the data.</param>
         /// <returns>A <see cref="Stream"/> containing the signed and encrypted data.</returns>
-        Stream Protect(Stream data, X509Certificate2 encryptionCertificate, X509Certificate2 signingCertificate);
-
+        Stream Protect(Stream data, X509Certificate2 encryptionCertificate, X509Certificate2 signingCertificate);
+
         /// <summary>
         /// Decrypts and then verifies the signature of the content in <paramref name="data"/>.
         /// </summary>
         /// <param name="data">A <see cref="Stream"/> containing the data which be decrypted and then the signature will be verified.</param>
-        /// <param name="signingCertificate">The public key <see cref="X509Certificate2"/> which will be used to validate the signature of the message data.</param>
+        /// <param name="signingCertificate">The public key <see cref="X509Certificate2"/> which will be used to validate the signature of the message data.</param>
+        /// <param name="logger"></param>
         /// <returns>A <see cref="Stream"/> containing the data in decrypted form.</returns>
-        Stream Unprotect(Stream data, X509Certificate2 signingCertificate);
+        Stream Unprotect(Stream data, X509Certificate2 signingCertificate, ILogger logger);
```

Comment:
Vi gikk bort fra dette patternet når vi gikk til 5.0. Loggeren skal inn via constructor.

Reply:
Skal være fiksa nå. Bare husket at det var sån det var gjort for config.